.circleci/modules-to-test.js

```diff
@@ -39,6 +46,31 @@ async function getModules(org, repo, prNumber) {
   }
 }
 
+async function modulesToSkip(org, repo, prNumber, branch) {
+  try {
+    const labelRes = await fetch(
+      `https://api.github.com/repos/${org}/${repo}/issues/${prNumber}/labels`
+    );
+    const labels = await labelRes.json();
+    const skipModules = branch.startsWith('email-service')
+      ? []
+      : ['fxa-email-service'];
+    if (Array.isArray(labels) && labels.find((l) => l.name === '🙈 skip ci')) {
```

Comment:
🙈 😆

Reply:
I thought the emoji might sort to the top of the label list, but no. still worth keeping though :)